api/CSSStyleSheet.json

```diff
@@ -62,7 +64,15 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "73",
```

Comment:
I can't find Firefox 73 in https://bugzil.la/1520690, but Chrome 73 is mentioned. Is this a mixup?

Reply:
that's the meta bug, it is in the experimental features list https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#constructable_stylesheets and there are a whole raft of individual issues starting with https://bugzilla.mozilla.org/show_bug.cgi?id=1604296

Reply:
I see, thanks! It ultimately doesn't matter too much if the version is wrong for features behind a flag, just thought this might be a mistake.